api/buildstatus/buildstatus_controller.go

```diff
@@ -0,0 +1,72 @@
+package buildstatus
+
+import (
+	"net/http"
+
+	"github.com/equinor/radix-api/api/utils"
+	"github.com/equinor/radix-api/models"
+	"github.com/gorilla/mux"
+)
+
+const rootPath = "/applications/{appName}"
+
+type buildStatusController struct {
+	*models.DefaultController
+}
+
+// NewBuildStatusController Constructor
+func NewBuildStatusController() models.Controller {
+	return &buildStatusController{}
+}
+
+// GetRoutes List the supported routes of this handler
+func (dc *buildStatusController) GetRoutes() models.Routes {
+	routes := models.Routes{
+		models.Route{
+			Path:        rootPath + "/buildstatus/{env}",
+			Method:      "GET",
+			HandlerFunc: GetBuildStatus,
+		},
+	}
+
+	return routes
+}
+
+// GetBuildStatus reveals build status for selected environment
+func GetBuildStatus(accounts models.Accounts, w http.ResponseWriter, r *http.Request) {
+	// swagger:operation GET /applications/{appName}/buildstatus/{env} application getBuildStatus
+	// ---
+	// summary: Show the application buildStatus
+	// parameters:
+	// - name: appName
+	//   in: path
+	//   description: name of Radix application
+	//   type: string
+	//   required: true
+	// - name: env
+	//   in: path
+	//   description: name of the environment
+	// responses:
+	//   "200":
+	//     description: "Successful operation"
+	//     schema:
```

Comment:
The route does not return an array of buildStatus. See if you can find a better response type here https://swagger.io/docs/specification/media-types/